db/seeders/letter_template_seeder.rb

```diff
@@ -92,7 +92,7 @@ def seed!
                   <p>
                   <br>
                   <br><strong>DATA PROTECTION ACT 2018: SUBJECT ACCESS REQUEST</strong>
-                  <br><strong><%= values.subject_full_name&.upcase %> - <%= values.prison_number&.upcase %><% if (values.prison_number.gsub(/[,]/, ' ').squeeze(' ').strip).match?(/[ ]/) %> [DELETE AS APPROPRIATE]<% end %></strong>
+                  <br><strong><%= values.subject_full_name&.upcase %> - <%= values.first_prison_number&.upcase %></strong>
```

Comment:
Why not `upcase` them in the method? would remove the need for the `&` syntax as we check for presence already there.

Reply:
Good call, have amended the method